pom.xml

```diff
@@ -13,17 +14,20 @@
   <packaging>maven-plugin</packaging>
 
   <name>maven-license-plugin Maven Plugin</name>
-  <url>http://maven.apache.org</url>
+  <url>https://maven.apache.org</url>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <sonatypeOssDistMgmtSnapshotsUrl>https://oss.sonatype.org/content/repositories/snapshots/</sonatypeOssDistMgmtSnapshotsUrl>
```

Comment:
I basically inlined most of what was in the last oss-parent release

Reply:
Well if we are going to use the Jenkins parent then delete this.